app/views/chapter/show.html.haml

```diff
@@ -2,14 +2,22 @@
   .row
     .large-12.columns
       %h1= @chapter.name
-      %p.lead
-        = t('chapters.intro', email: @chapter.email).html_safe
+  .row
+    .large-7.columns
+      - if @chapter.description
+        %p.lead= @chapter.description
+      - else
+        %p.lead= t('chapters.intro').html_safe
```

Comment:
@matyikriszta  there is a trick you can do with the locale to resolve the codeclimate warning. When a locale's name is suffixed with `_html` then you don't need to use `html_safe` in the views.

e.g. in this instance it should be changed to `chapters.intro_html`.

Reply:
@despo cool, I'll give that a try

Reply:
@despo unfortunately that trick doesn't solve the issue where it's actually reported: `t('chapters.info', @chapter.email).html_safe`, but I fixed it by manually escaping the model attribute with `h()`.